src/repositories/user-repository/UserRepository.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable, Inject } from '@graphql-modules/di';
+import { User } from './User';
+import { TableService, TableQuery } from 'azure-storage';
+
+export interface IUserRepository {
+  save(user: User): Promise<void>;
+  find(): Promise<User[]>;
+  tableName: string;
+}
+
+@Injectable()
+class UserRepository implements IUserRepository {
+  public tableName: string = 'userentity';
+
+  constructor(@Inject('TableService') private tableService: TableService) {
+    this.tableService.doesTableExist(this.tableName, (error, result) => {
```

Comment:
This needs to change- was just introduced as a temporary way way to ensure the resource exists. Is not optimal really. Needs to be moved to a separate area.

Reply:
cool, can you point me to an example of a better approach?

Reply:
I'll leave it like this for now, then we can move all table creating somewhere more appropriate?